contracts/socket/SocketDst.sol

```diff
@@ -164,43 +178,53 @@ abstract contract SocketDst is SocketBase {
         );
 
         (address executor, bool isValidExecutor) = executionManager__
-            .isExecutor(packedMessage, signature_);
+            .isExecutor(packedMessage, executionDetails_.signature);
         if (!isValidExecutor) revert NotExecutor();
 
         _verify(
-            packetId_,
-            proposalCount_,
+            executionDetails_.packetId,
+            executionDetails_.proposalCount,
             remoteSlug,
             packedMessage,
+            packetRoot,
             plugConfig,
-            messageDetails_.decapacitorProof,
+            bytes(""), //messageDetails_.decapacitorProof,
```

Comment:
why are we using empty bytes here? as part of socket, shouldnt we keep this generalized?

Reply:
good catch

Reply:
good catch

Reply:
thanks, fixed!